pom.xml

```diff
@@ -61,8 +61,10 @@
         <junit.version>5.9.0</junit.version>
         <lombok.version>1.18.30</lombok.version>
         <lombok-maven-plugin.version>1.18.20.0</lombok-maven-plugin.version>
-        <hadoop.version>3.4.0</hadoop.version>
+        <hadoop.version>3.3.6</hadoop.version>
```

Comment:
Why does this need to be downgraded? There are vulnerabilities in this older version

Reply:
Glue Sync implementation has introduced "aws sdk v2" dependency and with hadoop 3.4.0 looks like we need to explicitly add aws sdk v1 as well which is resulting in conflict.

Reply:
No longer seeing the issue with hadoop 3.4.1 and more recent version of aws sdk v2